client/views/room/Room/VoiceRoom.tsx

```diff
@@ -0,0 +1,112 @@
+import { Box, Button, ButtonGroup, Icon } from '@rocket.chat/fuselage';
+import { types } from 'mediasoup-client';
+import React, { FC, ReactElement, useState } from 'react';
+
+import VoiceRoomClient from '../../../../app/voice-channel/client';
+import { IRoom } from '../../../../definition/IRoom';
+
+interface IVoiceRoom {
+	room: IRoom;
+}
+
+let roomClient: VoiceRoomClient;
+
+const VoiceRoom: FC<IVoiceRoom> = (props): ReactElement => {
+	const { room } = props;
+
+	const [connected, setConnected] = useState(false);
+	const [muteMic, setMuteMic] = useState(false);
+
+	const handleJoin = async (): Promise<void> => {
+		roomClient = new VoiceRoomClient({
+			roomID: room._id,
+			device: {},
+			produce: true,
+			consume: true,
+			displayName: room.u.name || 'Anonymous',
+			peerID: room.u._id,
+		});
+
+		try {
+			await roomClient.join();
+			roomClient.on('newConsumer', (consumer: types.Consumer, peer) => {
```

Comment:
I don't think we should manipulate the DOM directly, we could replace it with React's approach

Reply:
Yes, I did this for testing purpose and to check the connectivity.
Will be creating the Peer/Avatar component replacing this.

Reply:
I got your point, let's move like this for now!